numba/cuda/cudadrv/driver.py

```diff
@@ -1780,6 +1787,52 @@ def auto_synchronize(self):
         yield self
         self.synchronize()
 
+    def add_callback(self, callback, arg):
+        """
+        Add a callback to a compute stream.
+        The user provided function is called from the driver thread once all
```

Comment:
There can be more than one driver thread, so it might be more accurate to state:

```suggestion
        The user provided function is called from a driver thread once all
```